h/static/scripts/controllers/search-bar-controller.js

```diff
@@ -4,6 +4,12 @@ var Controller = require('../base/controller');
 var LozengeController = require('./lozenge-controller');
 var AutosuggestDropdownController = require('./autosuggest-dropdown-controller');
 var SearchTextParser = require('../util/search-text-parser');
+var stringUtil = require('../util/string');
+var escapeHtml = require('escape-html');
```

Comment:
In both the Python and JS code, we follow a pattern of putting vendor imports at the top of the file, followed by a blank line, then local requires.